lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -767,7 +767,12 @@ CallInst *mutateCallInst(
                            InstName, TakeFuncName);
   NewCI->setDebugLoc(CI->getDebugLoc());
   NewCI->copyMetadata(*CI);
-  NewCI->setAttributes(CI->getAttributes());  
+  NewCI->setAttributes(CI->getAttributes());
```

Comment:
Is it a cherry-pick or a stand-alone fix?

Reply:
Also there were spaces at the end previously